auth/predicates/common.py

```diff
@@ -126,3 +127,9 @@ def fn(self, user: User):
         return set(education_group_types) & getattr(user, cache_key)
 
     return fn
+
+
+@predicate(bind=True)
+@predicate_failed_msg(message=_("Admission has been sent to EPC."))
+def is_sent_to_epc(self, user: User, obj: BaseAdmission):
```

Comment:
Comme discuté lors de votre venue; ceci devrait plutot être calculée dans la vue et checké dans le template; vois-tu ?
Cela ne devrait pas être un prédicat

Reply:
Je ne sais plus si on en a parlé ce matin ou hier matin avec juste Sylvain et Julien, mais on avait évoquer le fait de le faire en même temps que le refactoring. On a pas mal de choses qui se base sur les permissions au niveau de l'affichage notamment. On risque d'avoir un affichage bâtard si on fait la moitié en permission et l'autre en métier.

Si tu veux toujours que je le fasse comme ça, j'essayerai de passer partout mais pour le coup ça toucher presque toutes les permissions ici.

Reply:
ok pour moi

Reply:
a faire en meme temps que le refactoring dans ce cas
peut être mettre un todo quelque part sauf si c'est noté quelque part chez vous

Reply:
C'est dans les prédicats donc ça sera forcément fait avec. Sauf si tu parles d'un todo pour le refactoring en lui même ? On n'a pas fait de ticket pour ça déjà ?

Reply:
Je sais pas s'il y a un ticket de déjà créé je t'avoue